tooling/check/car.go

```diff
@@ -30,38 +32,38 @@ func decoded(cidStr string) cid.Cid {
 	return cid
 }
 
-func (c CheckIsCarFile) HasBlock(cidStr string) CheckIsCarFile {
+func (c CheckIsCarFile) HasBlock(cidStr string) *CheckIsCarFile {
```

Comment:
Do we want this too?
```suggestion
func (c *CheckIsCarFile) HasBlock(cidStr string) *CheckIsCarFile {
```

Reply:
(discussed out of band, we want to use values, and get clones)